js/components/graph/Graph.js

```diff
@@ -50,9 +50,10 @@ export default class Graph extends React.Component {
       infoBoxNodeId: "",
       panning: false,
       panStartX: 0,
-      panStartY:0,
+      panStartY: 0,
       showCourseModal: false,
-      showGraphDropdown:false
+      showGraphDropdown: false,
+      selectedNodes: []
```

Comment:
Make this a `Set` instead of an `Array`. In terms of display order when rendering inside the sidebar component, you should convert the set to an array, and then sort the array.